<launch>
    <arg name="test_recorder" default="true"/>

    <arg name="config_file" default="$(find ros_data_recorder)/config.yaml"/>
    <arg name="data_dir" default="/media/tl/Lenovo/data/ros_recordings"/>

    <node pkg="ros_data_recorder" type="ros_data_recorder_node" name="ros_data_recorder_node" respawn="true" output="screen">
        <param name="config_file" value="$(arg config_file)"/>
        <param name="data_dir" value="$(arg data_dir)"/>
    </node>

    <node pkg="ros_data_recorder" type="test_recorder" name="test_recorder" respawn="true" output="screen" if="$(arg test_recorder)"/>
</launch>